### DISCUSSION TITLE
Major phishing, why the single image in this spam doesn't score at all?

### DISCUSSION THREAD

**Discussion by flaviozluca** [rspamd/rspamd#4314] [Q&A]
Friends.

Major phishing, why the single image in this spam doesn't score at all?
[Res  Infoemail - Aviso Importante. - sac(ma@email.com.br) - 2022-10-19 0950.zip](https://github.com/rspamd/rspamd/files/9822809/Res.Infoemail.-.Aviso.Importante.-.sac.ma%40email.com.br.-.2022-10-19.0950.zip)

Thanks.

**Comment by flaviozluca** [comment]
1 more. pls, someone help me understand. Thanks.
[image spam 2.zip](https://github.com/rspamd/rspamd/files/9842138/image.spam.2.zip)

**Comment by flaviozluca** [comment]
1 more
[Purchase Succesful Id -Rfw-Rr6Hg5G-cv#dfg5 - 'Any _ Support Dept.' (www.ali323810@gmail.com) - 2022-12-05 1109.zip](https://github.com/rspamd/rspamd/files/10158271/Purchase.Succesful.Id.-Rfw-Rr6Hg5G-cv.dfg5.-.Any._.Support.Dept.www.ali323810%40gmail.com.-.2022-12-05.1109.zip)

**Comment by flaviozluca** [comment]
1 more
[more.zip](https://github.com/rspamd/rspamd/files/10168613/more.zip)

**Comment by fatalbanana** [comment]
Which rule(s) do you expect should match?

`HTML_SHORT_LINK_IMG_1` through `HTML_SHORT_LINK_IMG_3` are looking for embedded images with some particular dimensions in short HTML parts, where those images are inside an anchor tag.

Lack of embedding is a reason they don't match in some of these cases, lack of anchoring in others.

**Comment by flaviozluca** [comment]
Thanks for your reply, Fatalbanana.

Those scores seem low.

I'm not sure, all I know is we have tons of false negatives of major phishing and spam where 95%+ of the email is image. I think it should score, independently of being embedded or absolute...

Look at all the samples, besides this thread.

https://github.com/rspamd/rspamd/discussions/3985
https://github.com/rspamd/rspamd/discussions/4092

Thanks.

**Comment by flaviozluca** [comment]
Fatal banana,

Await your opinion on what I wrote.

Also, check out, 2 tests I did, absolute and embedded, none scored.
[Downloads.zip](https://github.com/rspamd/rspamd/files/10268635/Downloads.zip)

Thanks.
